jcapiv1/docs/System.md

```diff
@@ -25,6 +25,7 @@ Name | Type | Description | Notes
 **agent_version** | **String** |  | [optional] 
 **connection_history** | **Array&lt;Object&gt;** |  | [optional] 
 **ssh_root_enabled** | **BOOLEAN** |  | [optional] 
+**id** | **String** |  | [optional] 
```

Comment:
Do we want both `id` and `_id` here?

Reply:
yes. both are in the system model, not sure why we have 2 IDs but it's seems to be the same ID number used in both fields.